<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSHOutEditor</class>
 <widget class="QWidget" name="SSHOutEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelUser">
     <property name="text">
      <string>User</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="textUser"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>Password</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="textPassword">
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
   <widget class="QLabel" name="labePrivateKey">
    <property name="text">
     <string>Private Key</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
  </item>
  <item row="2" column="1">
   <widget class="QLineEdit" name="textPrivateKey"/>
  </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
